pkg/controller/storageversionmigrator/storageversionmigrator.go

```diff
@@ -318,3 +337,13 @@ func (svmc *SVMController) sync(ctx context.Context, key string) error {
 	logger.V(4).Info("Finished syncing svm resource", "key", key, "gvr", gvr.String(), "elapsed", time.Since(startTime))
 	return nil
 }
+
+type typeMetaUIDRV struct {
+	metav1.TypeMeta     `json:",inline"`
+	objectMetaUIDRVOnly `json:"metadata,omitempty"`
+}
+
+type objectMetaUIDRVOnly struct {
```

Comment:
nit: I was having trouble visually parsing `UID` and `RV` when concatenated, maybe:

```suggestion
type objectMetaUIDandRV struct {
```